File: src/nomanssave/cm.java
package nomanssave;

import java.io.File;
import javax.swing.Icon;
import javax.swing.filechooser.FileView;

class cm extends FileView {
   // $FF: synthetic field
   final cl fI;

   cm(cl var1) {
      this.fI = var1;
   }

   public Icon getIcon(File var1) {
      String var2 = var1.getName();
      return (Icon)(!var2.endsWith(".pb3") && !var2.endsWith(".pb0") ? super.getIcon(var1) : cl.as());
   }

   public String getName(File var1) {
      String var2 = var1.getName();
      return !var2.endsWith(".pb3") && !var2.endsWith(".pb0") ? var2 : var2.substring(0, var2.length() - 4);
   }
}

File: NMSSaveEditor/nomanssave/lower/cm.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{

class cm : FileView {
   cl fI;

   cm(cl var1) {
      this.fI = var1;
   }

   public Icon getIcon(FileInfo var1) {
      string var2 = var1.Name;
      return (Icon)(!var2.EndsWith(".pb3") && !var2.EndsWith(".pb0") ? base.getIcon(var1) : cl.@as());
   }

   public string getName(FileInfo var1) {
      string var2 = var1.Name;
      return !var2.EndsWith(".pb3") && !var2.EndsWith(".pb0") ? var2 : var2.Substring(0, var2.length() - 4);
   }
}

}
